### DISCUSSION TITLE
How to generate a report with a certain language?

### DISCUSSION THREAD

**Discussion by jeffjiang0613** [allure-framework/allure2#2470] [Questions & Support]
The default language of the generated report is english, but how can I switch it to other languages?

**Comment by baev** [comment]
Since [2.28.0](https://github.com/allure-framework/allure2/releases/tag/2.28.0), there is a new option available for that:

```bash
$ allure generate --lang de path/to/allure-results
```

See https://github.com/allure-framework/allure2/pull/2476
